src/ElmSharp/ElmSharp/Transit.cs

```diff
@@ -246,7 +246,18 @@ public double EndAccelerationFactor
         /// <since_tizen> preview </since_tizen>
         public void Go(double interval = 0)
         {
-            Interop.Elementary.elm_transit_go_in(_handle, interval);
+            Double timeInterval = interval;
+
+            if (timeInterval.Equals(0.0))
```

Comment:
```suggestion
            if (timeInterval == 0)
```

Reply:
Thank you for review. I have updated the patch.